texcount: add page (#10625)
tldr-pages/tldr: pages/common/texcount.md

# texcount

> > Count words in TeX documents omiting macros.
> > More information: <https://app.uio.no/ifi/texcount/index.html>.

Reviewer: ```suggestion
> Count words in TeX documents omitting macros.
> More information: <https://app.uio.no/ifi/texcount/howto.html>.
```

> - Count words in one TeX file:
> 
> `texcount path/to/file`

Reviewer: ```suggestion
- Count words in a TeX file:

`texcount {{path/to/file.tex}}`
```

> - Count words in document and subdocuments built with \input or \include:
> 
> `texcount -merge file`

Reviewer: ```suggestion
- Count words in a document and subdocuments built with `\input` or `\include`:

`texcount -merge {{path/to/file.tex}}`
```

Author: So the problem is that when the program `texcount -merge` or `texcount -inc` is called and the root TeX file includes or inputs other files, the program call needs to be from the root TeX file directory. Otherwise it doesn't find the linked files. At least when I tested it this behaviour was the case.

Any idea how to make this clear?

Reviewer: > So the problem is that when the program `texcount -merge` or `texcount -inc` is called and the root TeX file includes or inputs other files, the program call needs to be from the root TeX file directory. Otherwise it doesn't find the linked files. At least when I tested it this behaviour was the case.
> 
> Any idea how to make this clear?

I added a notice to the description and changed the affected placeholders from `path/to/file.tex` to `file.tex`.

> - Count words in document and subdocuments list each file extra:
> 
> `texcount -inc file`

Reviewer: ```suggestion
- Count words in a document and subdocuments, listing each file separately (and a total count):

`texcount -inc {{path/to/file.tex}}`
```

Author: So the problem is that when the program `texcount -merge` or `texcount -inc` is called and the root TeX file includes or inputs other files, the program call needs to be from the root TeX file directory. Otherwise it doesn't find the linked files. At least when I tested it this behaviour was the case.

Any idea how to make this clear?

- Count words with verbose output:

> `texcount -v path/to/file`

Reviewer: ```suggestion
- Count words with verbose output:

`texcount -v {{path/to/file.tex}}`
